imports/plugins/core/catalog/server/no-meteor/schemas/schema.graphql

```diff
@@ -27,6 +45,18 @@ type PriceRange {
   range: String!
 }
 
+"Represents a name of a filter to be applied to the Catalog collection"
+input CatalogFilter {
```

Comment:
Currently GraphQL does not support input unions, and we'll definitely need non-bool filtering eventually. So I think I'd call this `CatalogFilterBoolean` and change the input field to `booleanFilters: [CatalogFilterBoolean]`. That way we can add `stringFilters`, etc. later.